programming/index.md

```diff
@@ -22,6 +22,10 @@ Our code is versioned using [Git](https://git-scm.com/). Git lets us collaborate
 - [Brief Introduction](https://try.github.io)
 - [Branching and Code Review](branching)
 
+## Vision Tracking
+How to use the opencv library and the basics of vision tracking and camera callibration
+- [Installation](https://github.com/Pigmice2733/wiki/blob/octoflo-installation/programming/opencv-installation.md)
```

Comment:
Can you use a relative link (like we've done above) rather than an absolute link here? Generally, when linking within a site, a relative link is better.